trino/dbapi.py

```diff
@@ -742,7 +742,7 @@ def execute(self, operation, params=None):
         self._query = trino.client.TrinoQuery(self._request, query=operation,
                                               legacy_primitive_types=self._legacy_primitive_types,
                                               fetch_mode="segments")
-        self._iterator = iter(self._query.execute())
+        self._iterator = map(lambda tuple: tuple[0], iter(self._query.execute()))
```

Comment:
Note this is a breaking change, but i agree it is a nicer API. 

However, how can the client retrieve the encoding?